finalProject/fizzBuzzChallenge.js

```diff
@@ -0,0 +1,7 @@
+for (let i = 1; i <= 25; i++) {
```

Comment:
I think you may have misinterpreted the directions on this challenge - it appears that this only works for numbers 0 - 25. How could you use a function/parameters to make this more dynamic and able to accept ANY range of numbers?